pkg/util/client/client.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package client
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func CreatePatch(before, after client.Object) (client.Patch, error) {
```

Comment:
can you remove the similar function in the `workload` package?

Reply:
You mean this one https://github.com/kubernetes-sigs/kueue/commit/dddbf3db029beeee63a7417b294b2393fa5f7e5c? I didn't found it.

Reply:
oh sorry, @gabesaba just experimented with it but we didn't merge it :)